packages/headless/src/features/product-listing/v2/product-listing-v2-slice.test.ts

```diff
@@ -13,12 +16,24 @@ describe('product-listing-v2-slice', () => {
   beforeEach(() => {
     state = getProductListingV2InitialState();
   });
+
   it('should have an initial state', () => {
     expect(productListingV2Reducer(undefined, {type: 'foo'})).toEqual(
       getProductListingV2InitialState()
     );
   });
 
+  it('should allow to set the product recommendations skus', () => {
```

Comment:
You marked this one as resolved without making any change, however unless I misunderstand what this test is about, it seems to me like the description should be something like "should allow to set the product listing url"

Reply:
Turns out the PLP v1 implementation had a test with the same faulty name, and I fixed that test name instead of this one which you commented on. 🥴 

Should be good now! 😅